python/ray/rllib/utils/sampler.py

```diff
@@ -132,6 +134,34 @@ def get_metrics(self):
         return completed
 
 
+def collect_samples(agents, min_samples):
```

Comment:
Let's just keep this where it is in rollout.py, we're going to remove it next anyways.